src/transport-layer.service.ts

```diff
@@ -0,0 +1,58 @@
+import axios, { AxiosRequestConfig, AxiosInstance } from 'axios';
+import { IServerConfiguration } from './shared/interfaces';
+import ITransportLayer from './transport.interface';
+
+const config: IServerConfiguration = {
+    baseURL: 'http://127.0.0.1:3337',
+    timeout: 100000,
+    headers: {},
+};
```

Comment:
Please in future create config util file. And store all configs for your application in this file. For configs centralization.